dkms.conf

```diff
@@ -1,7 +1,5 @@
 PACKAGE_NAME="rtl8812AU"
-PACKAGE_VERSION="#MODULE_VERSION#"
+PACKAGE_VERSION="4.3.22"
```

Comment:
I *think* I would, but I don't know whether 'your' or 'mine' would be better. Maybe others can chime in on this one.

What I really would like is that you'd split your changes up into [atomic commits](https://github.com/diederikdehaas/rtl8812AU/pull/22#issuecomment-248797675).
All the changes from `:=` to `?=` are a nice fit for 1 commit, the use of `KERNEL_RELEASE` into another, the removal of the `if...else...endif` construct as a third and the changes to the `dkms.conf` as a forth commit.

That way if it turns out one of these changes has an unwanted side-effect, I can revert that change without reverting all the changes you have proposed here.